frontend/src/app/login/page.tsx

```
@@ -0,0 +1,119 @@
+                  </label>
+                  <input
+                    type="email"
+                    placeholder="email"
+                    className="input secondary-theme dark-font w-full"
+                    required
+                  />
+                </div>
+                <div className="">
+                  <label className="label">
+                    <span className="label-text">Password</span>
+                  </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)